README.md

```diff
@@ -508,11 +518,11 @@ Authentication tokens are passed in on the `context.auth` object. If no authenti
 
 ### Action Retries
 
-If you expect that your action might hit the 30 second time limit, consider adding retry support with `context.retry`. You **must** return this in order for your action to retry- it will not do so automatically if it hits the timeout. See a full action example using this [here](https://github.com/clay-run/action-package-apis/blob/master/src/aws_textract_pdf/aws_textract_pdf.js#L81).
+If you expect that your action might hit the 30 second time limit, consider adding retry support with `context.retry`. You **must** return this in order for your action to retry- it will not do so automatically if it hits the timeout. See a full action example using this [here](https://github.com/clay-run/action-package-apis/blob/master/src/aws_textract_pdf/aws_textract_pdf.js#L71-L74).
```

Comment:
I know you didn't write this comment below, but same comment as for the clay-action-client: we should really think about separating out documentation that is internal vs documentation for our users. users will not be able to access the action-package-apis repo

Reply:
Ah I see. That is a good point.

Reply:
I'll create a clubhouse ticket to keep this point in mind

Reply:
https://app.clubhouse.io/clay-run/story/3600/separating-out-documentation-that-is-internal-vs-documentation-for-our-users